plugins/modules/aci_l3out_extepg.py

```diff
@@ -305,22 +377,43 @@ def main():
             module_object=extepg,
             target_filter={"name": extepg},
         ),
+        child_classes=["fvRsSecInherited", "l3extRsInstPToProfile"],
     )
 
     aci.get_existing()
 
     if state == "present":
-        aci.payload(
-            aci_class="l3extInstP",
-            class_config=dict(
-                name=extepg,
-                descr=description,
-                prefGrMemb=preferred_group,
-                targetDscp=dscp,
-                nameAlias=name_alias,
-            ),
+        child_configs = []
+        if contract_masters:
+            for contract in contract_masters:
```

Comment:
Do we want to support deletion of child objects from this module? Applies to contract_masters and route_control_profile.

Reply:
We decided to implement a separate module for contract masters which will be part of: https://github.com/CiscoDevNet/ansible-aci/issues/126

We decided to slightly modify the structure of the route control profiles which will be part of my next commit